onebusaway-android/src/main/java/org/onebusaway/android/report/ui/dialog/RegionValidateDialog.java

```diff
@@ -0,0 +1,59 @@
+/*
+* Copyright (C) 2014 University of South Florida (sjbarbeau@gmail.com)
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.onebusaway.android.report.ui.dialog;
+
+import org.onebusaway.android.R;
+import org.onebusaway.android.app.Application;
+import org.onebusaway.android.report.ui.ReportActivity;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+/**
+ * Show to validate if the current user is in expected region
+ *
+ * @author Cagri Cetin
+ */
+public class RegionValidateDialog extends BaseReportDialogFragment {
+
+    @Override
+    @NonNull
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+
+        StringBuilder message = new StringBuilder();
+        String regionName = Application.get().getCurrentRegion().getName();
+        message.append(getResources().getString(R.string.region_dialog_Message2, regionName)).append("?");
+
+        AlertDialog dialog =  new AlertDialog.Builder(getActivity())
+                .setMessage(message.toString())
+                .setPositiveButton("YES", new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int which) {
+                        ((ReportActivity) getActivity()).createIssueTypeListFragment();
+                    }
+                })
+                .setNegativeButton("NO", new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int which) {
+                        closeSuperActivity();
+                        ((ReportActivity) getActivity()).createPreferencesActivity();
```

Comment:
@cagryInside This transition is currently a little abrupt for users if they answer "no".  For example, assume I am physically in Seattle.  I tap "Send feedback", and the app asks me (for a reason unknown to the user) "Are you in Tampa Bay?"  I tap "No", and suddenly I'm looking at the Preferences.  I don't know why.  I tap back to go back to submitting an issue, and go through all this again.

Let's add a dialog in the PreferenceActivity that is only shown if the user is directed to the PreferencesActivity from this screen (we can pass a variable via the Intent to trigger this).  It should have the title of "Check Region", with a single "Ok" button, and the text "Your region is currently set to (regionName), but you said you weren't located in (regionName).  Please tap on 'Your Region" and change to the correct region".